gammapy/data/data_store.py

```diff
@@ -304,6 +306,25 @@ def check_available_event_lists(self, logger=None):
 
         return file_available
 
+    def create_new_store_from_obs_table(self, obs_table, outdir):
+        """Create a new `~gammapy.data.DataStore` containing a subset of observations
+
+        Parameters
+        ----------
+        obs_tabe : `~gammapy.data.ObservationTable`
+            Table of observation to create the subset
+        dir : str, Path
+            Directory for the new store
+        """
+        outdir = make_path(outdir)
+        obs_ids = obs_table['OBS_ID'].data
+        hdutable = self.hdu_table
+        hdutable.add_index('OBS_ID')
+        temp = hdutable.loc[obs_ids]
+        copydirs = set(temp['FILE_DIR'])
```

Comment:
For now I think this is OK.

For the future, it might be nice to re-introduce "schemes" how to store data and implement them.
E.g. one HDU per file, one obs per file, or all obs in one file with very many HDUs.

But this is just nice to have ... for now +1 to do it this way and move on ...